edit/logging-flow/index.vue

```diff
@@ -199,12 +200,16 @@ export default {
     filtersYaml: {
       deep: true,
       handler() {
-        const filterJson = jsyaml.load(this.filtersYaml);
+        try {
+          const filterJson = jsyaml.load(this.filtersYaml);
 
-        if ( isArray(filterJson) ) {
-          set(this.value.spec, 'filters', filterJson);
-        } else {
-          set(this.value.spec, 'filters', undefined);
+          if ( isArray(filterJson) ) {
+            set(this.value.spec, 'filters', filterJson);
+          } else {
+            set(this.value.spec, 'filters', undefined);
```

Comment:
I'm wondering if we should set `filters` to `undefined` when an error is raised instead of displaying the error. I don't think any other yaml editors behave like this in rancher dashboard.